2.PyGMT_ecosystem.py

```diff
@@ -0,0 +1,63 @@
+import pygmt
+
+red, blue = "238/86/52", "#4B8BBE"  # GMT red and Python blue
+
+fig = pygmt.Figure()
+fig.basemap(region=[-1, 16, -1, 8], projection="x1c", frame="+n")
+
+# Add the GMT part.
+fig.plot(x=[-0.5, 2.5, 2.5, -0.5], y=[0, 0, 6, 6], pen="1p,black", close=True)
+fig.vlines(x=1, ymin=1, ymax=3, pen="1p,black")
+fig.vlines(x=1, ymin=3, ymax=5, pen="1p,black")
+fig.plot(x=[1.0] * 3, y=[1, 3, 5], style="R2.1c/1c/3p", fill=red, pen="0.5p,black")
+fig.text(
+    x=[1.0] * 4,
+    y=[1.25, 0.75, 3, 5],
+    text=["GMT low-level", "library", "GMT C API", "GMT modules"] ,
+    font="8p,1,white",
+)
+fig.logo(position="g1/7+jMC+w2c")
+
+# Add the Python part.
+fig.plot(x=[7.5, 15.5, 15.5, 7.5], y=[0, 0, 6, 6], pen="1p,black", close=True)
+for x, y in [
+    ([10, 14], [1, 1]),
+    ([10, 9], [1, 3]),
+    ([9, 8.7], [3, 5]),
+    ([9, 10.25], [3, 5]),
+    ([10, 12], [1, 3]),
+    ([12, 12], [3, 5]),
+]:
+    fig.plot(x=x, y=y, pen="1p,black")
+fig.plot(x=[10, 9, 12], y=[1, 3, 3], style="R2.0c/1c/3p", fill=blue, pen="0.5p,black")
+fig.plot(x=14, y=1, style="R2.0c/1c/3p", fill=blue, pen="0.5p,black,-")
+fig.text(
+    x=[10, 9, 12, 14],
+    y=[1, 3, 3, 1],
+    text=["NumPy", "Xarray", "Pandas", "PyArrow"],
+    font="8p,1,white",
+)
+fig.text(
+    x=[8.7, 10.25, 12],
+    y=[5, 5, 5],
+    text=["rioxarray", "contextily", "GeoPandas"],
+    font="8p,1,white",
+    fill=blue,
+    pen="0.5p,black,-",
+    clearance="0.1c/0.1c+tO",
+)
+fig.image(
+    "https://www.python.org/static/community_logos/python-logo-master-v3-TM.png",
+    position="g12/7+jMC+w4.5c"
```

Comment:
Would place the Python logo relative to the horizontal middle of the box:

| `g12/7` | `g11.5/7` |
| -- | -- |
| <img width="1894" height="919" alt="paper_pygmt_fig2_ecosyst_posold" src="https://github.com/user-attachments/assets/06c89e06-371d-4f84-81d0-bb8db4739d5c" /> | <img width="1894" height="919" alt="paper_pygmt_fig2_ecosyst_posnew" src="https://github.com/user-attachments/assets/ac25d70c-16be-4e7f-a279-2eeb9bf332a7" /> |

```suggestion
    position="g11.5/7+jMC+w4.5c"  # 7.5 + (15.5 - 7.5)
```